en/asgardeo/docs/apis/organization-apis/restapis/notification-senders-v2.yaml

```diff
@@ -0,0 +1,1346 @@
+openapi: 3.0.0
+info:
+  title: Notification Senders API
+  description: This document specifies **Email, SMS and Push notification sender RESTful API** for **Asgardeo** organizations.
+               To access the notification sender organization APIs in Asgardeo, you need to first get an access token from your organization.
+  version: "v2"
+servers:
+  - url: https://api.asgardeo.io/t/{root-organization-name}/o/api/server/v2
+    variables:
+      serverUrl:
+        default: https://api.asgardeo.io/t/{root-organization-name}/o/
+      root-organization-name:
+        default: "{root-organization-name}"
+security:
+  - OAuth2: []
+paths:
+  /notification-senders/email:
+    get:
+      tags:
+        - Email Senders
+      summary: Get a list of email senders
+      description: |
+        This API provides the capability to retrieve the list of email senders. <br>
+          <b>Permission required:</b> <br>
```

Comment:
Are permissions applicable for b2b orgs?

Reply:
updated permissions